core/server/master/src/main/java/alluxio/master/meta/MetaMasterInfo.java

```diff
@@ -22,7 +22,7 @@
  * Master information.
  */
 @NotThreadSafe
-public final class MasterInfo {
+public final class MetaMasterInfo {
```

Comment:
why did we need this rename?

Reply:
Package Wire need a MasterInfo class, and I want to make a difference with it. Also package meta has a lot of MetaXXX class, so I change this class name.

Reply:
Actually in the package `meta` all those are `MetaMaster` which correspond to the `MetaMaster` class so `MetaMaster` is one word. This `MasterInfo` actually means the metadata of masters so `Meta-Master` actually are two separate words. Therefore I prefer this stays `MasterInfo`. Let's keep the old name. Thanks!

Reply:
OK, I will fix that.